### PR TITLE
Add clients page and form

### PR DESCRIPTION
**Pull request by naomiaz**

### CODE REVIEW

File: components/clients-overview/ClientsOverview.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              New Client
+            </button>
+          </Link>
+
+          <select
+            className="input-field clients-overview__filter"
+            name="sortBy"
+            value={sortBy}
+            id="select"
+            onChange={({ target }) => (target.value) && onSortClientsBy(target.value)}
```
Comment (boris-arkenaar) [inline components/clients-overview/ClientsOverview.jsx]: Dit ziet er verwarrend uit, ik weet niet precies hoe dit uitgevoerd zal worden. Je bedoelt waarschijnlijk
```
onChange={({ target }) => (target.value && onSortClientsBy(target.value))}
```

File: components/clients-overview/ClientsOverview.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <option value="ascending">A-Z (asc)</option>
+            <option value="descending">Z-A (desc)</option>
+          </select>
+        </div>
+
+        <ul>
+          {clients && clients.map((currentClient, index) => (
+            <li key={currentClient.id} id={currentClient.id} className="client-item">
+              <ClientItem
+                index={index}
```
Comment (boris-arkenaar) [inline components/clients-overview/ClientsOverview.jsx]: `index`?

File: ducks/clients.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      const clientA = a[sortBy].toUpperCase();
+      const clientB = b[sortBy].toUpperCase();
+      if (clientA < clientB) { return sortDirection === 'ascending' ? -1 : 1; }
+      if (clientA > clientB) { return sortDirection === 'ascending' ? 1 : -1; }
+      return 0;
+    })
+  )
+);
+
+export const isFormSavingSelector = (state) => state.clients.isFormSaving;
```
Comment (boris-arkenaar) [inline ducks/clients.js]: Why not use the `clientsRootSelector` here as well?

File: components/client-form/client-form.scss [hunk 4]
```diff
@@ -220,27 +225,21 @@
...
-    display: flex;
-    flex-flow: row;
-    justify-content: space-between;
-  }
-
-  &__first-name,
-  &__zip {
-    margin-right: 10px;
-  }
+  // &__name,
```
Comment (boris-arkenaar) [inline components/client-form/client-form.scss]: Remove this.

File: components/client-item/ClientItem.jsx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+                {clientName}
+              </span>
+              <span className="client-item__text--secondary">
+                {city}
+              </span>
+            </div>
+          </div>
+
+          {/* WORK INFO */}
+          <ul className="client-item__client-wrapper">
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply components/client-item/ClientItem.jsx]: This is to maintain consistency, as the first wrapper is called `personalia-wrapper` (which also includes a `personalia` class)

File: components/clients-overview/ClientsOverview.jsx (hunk 1 above)
Comment (naomiaz) [reply components/clients-overview/ClientsOverview.jsx]: Correct!

File: components/clients-overview/ClientsOverview.jsx (hunk 2 above)
Comment (naomiaz) [reply components/clients-overview/ClientsOverview.jsx]: Removed it, can't remember why I put it there..

File: components/client-item/ClientItem.jsx [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+                </a>
+              </span>
+              <span className="client-item__text--secondary">
+                Website
+              </span>
+            </li>
+          </ul>
+
+          {/* CARET */}
+          <div className={`client-item__button-caret client-item__button-caret${isDetailWrapperVisible ? '--up' : '--down'}`} />
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply components/client-item/ClientItem.jsx]: Fixed this

File: components/client-form/ClientForm.jsx [hunk 7]
```diff
@@ -97,15 +95,17 @@ class ClientForm extends React.Component {
             <Link href="/clients">
               <a className="render-whitespace--left">
                 <button
-                  className="btn client-form__button client-form__button--cancel"
+                  className="btn client-form__button
+                  client-form__button--cancel"
```
Comment (boris-arkenaar) [inline components/client-form/ClientForm.jsx]: At least indent this line, but I'd prefer to see something like
```
className="
  btn
  client-form__button
  client-form__button--cancel
"

Comment (naomiaz) [reply components/client-form/ClientForm.jsx]: Fixed this!

**Pull request merged**